exercises/practice/yacht/.meta/example.ex

```diff
@@ -0,0 +1,76 @@
+defmodule Yacht do
+  @doc """
+  Calculate the score of the list of 5 dice rolls using the given category.
+  """
+  @spec score(category :: String.t(), dice :: [integer]) :: integer
+  def score(category, dice)
+
+  def score("ones", dice), do: score(1, dice)
+  def score("twos", dice), do: score(2, dice)
+  def score("threes", dice), do: score(3, dice)
+  def score("fours", dice), do: score(4, dice)
+  def score("fives", dice), do: score(5, dice)
+  def score("sixes", dice), do: score(6, dice)
+
+  def score(number, dice) when is_integer(number) do
+    Enum.count(dice, &(&1 == number)) * number
+  end
+
+  def score("full house", dice) do
+    full_house =
+      Enum.frequencies(dice)
```

Comment:
This function is only available from Elixir 1.10. One of the challenges of the example solution is that it needs to work in older versions too, currently we start testing from 1.7. I'm going to update the documentation to make this more explicit.

We had this problem with another exercise already 🙂 here:
https://github.com/exercism/elixir/blob/0caae7f55f150e0af192d7720eb0c10907572ae5/exercises/practice/word-count/.meta/example.ex#L27-L33

Reply:
Got it, I'll update my solution to not use `Enum.frequencies/1` and will test it on Elixir 1.7 to make sure it works with that as well.